lago/cmd.py

```diff
@@ -398,6 +400,44 @@ def do_console(prefix, host, **kwargs):
 
 
 @lago.plugins.cli.cli_plugin(
+    help='Create Ansible hosts file of the environment'
+)
+@in_lago_prefix
+@with_logging
+def do_generate_ansible_hosts(prefix, out_format, **kwargs):
+    """
+    This method iterate through all the VMs and create an ansible
+    hosts configuration. It defines an IP address and private key
+    for the machine in group named by its type, it is printed to
+    standard output. The content looks for example like this:
+
+    [ovirt-host]
+    1.2.3.4 ansible_host=1.2.3.4 ansible_ssh_private_key_file=/path/id_rsa
+    1.2.3.6 ansible_host=1.2.3.6 ansible_ssh_private_key_file=/path/id_rsa
+    [ovirt-engine]
+    1.2.3.5 ansible_host=1.2.3.5 ansible_ssh_private_key_file=/path/id_rsa
+    """
+    inventory = defaultdict(list)
+    for vm in prefix.virt_env.get_vms().values():
+        vm_mgmt_ip = [
+            nic.get('ip') for nic in vm.nics()
+            if 'management' in nic.get('net')
```

Comment:
This prints nothing for me, I think the management properties are in the ``prefix.get_nets()``, and then you can use ``nets['net_name'].is_management()``.